src/main/java/uk/gov/hmcts/reform/pdf/service/service/AuthorisationService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.pdf.service.service;
+
+import feign.FeignException;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.pdf.service.client.S2sClient;
+import uk.gov.hmcts.reform.pdf.service.exception.AuthorisationException;
+
+@Service
+public class AuthorisationService {
+
+    public static final String SERVICE_AUTHORISATION_HEADER = "ServiceAuthorization";
+
+    private final S2sClient s2sClient;
+
+    public AuthorisationService(S2sClient s2sClient) {
+        this.s2sClient = s2sClient;
+    }
+
+    /**
+     * Authenticates the service.
+     * Returns service name on success.
+     */
+    public String authorise(String serviceAuthHeader) {
```

Comment:
Where is this return value used?

Reply:
nowhere. just reusing implementation like other services do. can silence it as only need to verify authenticity

Reply:
I'd change the return type to void -- unused code should be removed.